pkg/devicetemplateconverter/devicetemplateconverter.go

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package devicetemplateconverter provides device template services.
+package devicetemplateconverter
+
+import (
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"go.thethings.network/lorawan-stack/pkg/component"
+	"go.thethings.network/lorawan-stack/pkg/devicetemplates"
+	"go.thethings.network/lorawan-stack/pkg/errors"
+	"go.thethings.network/lorawan-stack/pkg/ttnpb"
+	"google.golang.org/grpc"
+)
+
+// Config represents the DeviceTemplateConverter configuration.
+type Config struct {
+	Enabled []string `name:"enabled" description:"Enabled converters"`
+}
+
+// DeviceTemplateConverter implements the Device Template Converter component.
+//
+// The Device Template Converter exposes the EndDeviceTemplateConverter service.
+type DeviceTemplateConverter struct {
+	*component.Component
+
+	converters map[string]devicetemplates.Converter
+
+	grpc struct {
```

Comment:
What is the reason for this 1 field struct ? Will it be expanded in the future ?

Reply:
Yes, may be, also for consistency with other components.